client-v2/src/actions/Collections.ts

```diff
@@ -179,11 +205,27 @@ function getCollections(
           id
         })
       );
+
+      let missingCollectionActions: Action[][];
```

Comment:
Decided using let was more readable than a reduce.

Reply:
Agreed, it felt like a filter -> map 👍